.github/copilot-instructions.md

```diff
@@ -82,6 +82,9 @@ When working on an issue:
   - [ ] Number ordered lists as "1." for every item (don't use sequential numbers)
   - [ ] Use backticks around content specifically for file names (`file.txt`), folders (`folder`), file paths (`folder/file.txt`), custom types (`myVariable`, `MyClass`), raw URLs in the text (`https://www.contoso.com`), URL segments (`/product/id/199`), file extensions (`.razor`), NuGet packages (`Microsoft.AspNetCore.SignalR.Client`), and code that should never be localized
   - [ ] For Blazor's Razor components mentioned in article text, use backticks around the name of the component (example: `Counter` component)
+  - [ ] Use placeholders with braces format: `{placeholderName}` consistently throughout the document when used in URIs, code examples, and other contexts where placeholders are needed
+    - [ ] Example: `https://localhost:{port}/openapi/v1.json` where the `{port}` placeholder represents the port number
```

Comment:
```suggestion
   ```diff
   - Launch the app and navigate to `https://localhost:<port>/openapi/v1.json` to view the generated OpenAPI document.
   + Launch the app and navigate to `https://localhost:{PORT}/openapi/v1.json` to view the generated OpenAPI document, where the `{port}` placeholder is the port.
   ```
```